gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/taskexecution/GradleFix.java

```diff
@@ -61,17 +63,34 @@ public SuggestedFix render(GradleFixContext context, VisitorState state) {
 
         SuggestedFix.Builder fixBuilder = SuggestedFix.builder();
 
-        // Replace the illegal call, taking care to preserve any receivers
+        // Preserve arguments (if any)
+        String fixedCallChain;
+        if (fixTemplate.numFormatArgs() == 1) {
+            String args = context.illegalCallToReplace.getArguments().stream()
+                    .map(state::getSourceForNode)
+                    .collect(Collectors.joining(", "));
```

Comment:
this seems weird? what are you joining here? should it not be an array?

If so, then it feels like there should be no difference between checking `numFormatArgs == 1` vs not.